WordPress/WordPressUITests/Tests/MainNavigationTests.swift

```diff
@@ -17,26 +17,41 @@ class MainNavigationTests: XCTestCase {
         removeApp()
     }
 
-    func testTabBarNavigation() throws {
+    // We run into an issue where the People screen would crash short after loading.
+    // See https://github.com/wordpress-mobile/WordPress-iOS/issues/20112.
+    //
+    // It would be wise to add similar tests for each item in the menu (then remove this comment).
+    func testLoadsPeopleScreen() throws {
         XCTAssert(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
 
-        _ = try mySiteScreen
-            .tabBar.goToReaderScreen()
+        mySiteScreen.goToMenu()
```

Comment:
Looks like it's failing on iPad because of this line. Since on iPad, the menu items are already listed on screen (including the "People" item), this isn't required anymore. What do you think about adding a condition for iPhone?

```suggestion
        if XCUIDevice.isPhone {
            mySiteScreen.goToMenu()
        }
```